library/options.inc.php

```diff
@@ -853,6 +853,13 @@ class='form-control{$smallform}'
         } else {
             echo "</select>";
         }
+        
+        //include here weno pharmacy selector once weno is enabled
+        if ($GLOBALS['weno_rx_enable']) {
+            echo "<br />";
+            include "templates/pharmacy_list_form.php";
+            echo "<br />";
+        }
```

Comment:
another place for an event listener, so can move this to module

Reply:
@bradymiller, @robertdown mentioned last year that he was going to work on this but nothing has been done so far or atleast reached out. Do you have any idea in the codebase where this has been implemented so that i can replicate? That would be helpful. @sjpadgett can you help or point me to a specific place?